karate-tests/src/test/java/mocks/sandbox/get-patient-search.js

```diff
@@ -125,21 +135,28 @@ if (request.pathMatches('/Patient') && request.get) {
   response.headers = basicResponseHeaders(request)
 
   const family = request.param('family')
-  const given = request.params.given
+  const given = request.params.given || []
   const gender = request.param('gender')
-  const birthDate = request.params.birthdate
+  const birthDate = request.params.birthdate || []
   const postalCode = request.param('address-postalcode')
   const fuzzyMatch = request.paramBool('_fuzzy-match')
   const phone = request.param('phone')
   const email = request.param('email')
   const maxResults = request.param('_max-results')
+  const historyMatch = request.param('_history')
+  const gp = request.param('general-practitioner')
+  const deathDate = request.param('death-date')
 
   if (validateHeaders(request) && validateQueryParams(request)) {
     if (fuzzyMatch) {
-      if (family === 'Garton' && given[0] === 'Bill' && birthDate[0] === '1946-06-23') {
+      if (family === 'Garton' && (given[0]) === 'Bill' && birthDate[0] === '1946-06-23') {
```

Comment:
this if/else might be an easier read as a switch statement
https://www.w3schools.com/js/js_switch.asp

but I'm not going to mandate it, see if you think it's feasible

Reply:
Agree, I am planning to create a separate task to make some improvements on mocks, I will try to implement suggested change as part of that ticket